sdk/src/Services/ElasticMapReduce/Custom/Model/StreamingStep.cs

```diff
@@ -30,7 +30,7 @@ namespace Amazon.ElasticMapReduce.Model
     ///
     /// HadoopJarStepConfig config = new StreamingStep {
     ///     Inputs = new List&lt;string> { "s3://elasticmapreduce/samples/wordcount/input" },
-    ///     Output = "s3://my-bucket/output/",
+    ///     Output = "s3://amzn-s3-demo-bucket/output/",
```

Comment:
Looks like the dry-run failed because this PR is changing `ElasticMapReduce` but the service was not included in the DevConfig.